src/state/slices/stakingDataSlice/selectors.ts

```diff
@@ -72,6 +73,42 @@ export const selectTotalStakingDelegationCryptoByAccountSpecifier = createSelect
   }
 )
 
+export const selectTotalStakingDelegationCrypto = createSelector(
+  selectStakingData,
+  // We make the assumption that all delegation rewards come from a single denom (asset)
+  // In the future there may be chains that support rewards in multiple denoms and this will need to be parsed differently
+  stakingData => {
+    let total = bnOrZero(0)
+
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    for (const [_, value] of Object.entries(stakingData.byAccountSpecifier)) {
+      const amount = reduce(
+        value?.delegations,
+        (acc, delegation) => acc.plus(bnOrZero(delegation.amount)),
+        bn(0)
+      )
+      total = total.plus(amount)
+    }
+
+    return total.toString()
```

Comment:
please don't use this let and mutate pattern, this can be done in a reduce. immutability >>> mutability